.eslintrc.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
     'prettier/@typescript-eslint'
   ],
   rules: {
-    '@typescript-eslint/no-floating-promises': 'warn',
+    '@typescript-eslint/no-floating-promises': 'error',
```

Comment:
Is `error` the default? Can these rules be removed?

Reply:
pretty sure that this particular rule is `off` by default